docs/tutorial.md

```diff
@@ -54,19 +53,19 @@ spec:
     name: echo-hello-world
 ```
 
-To apply the yaml files, use the following command:
+Apply your YAML files to the build as follows:
 
 ```bash
 kubectl apply -f <name-of-file.yaml>
 ```
 
-To see the output of the `TaskRun`, use the following command:
+To check whether running your `Task` succeeded, use the following command:
 
 ```bash
 tkn taskrun describe echo-hello-world-task-run
 ```
 
-You will get an output similar to the following:
+Below is the example output:
```

Comment:
```suggestion
You will get output similar to the following:
```

Reply:
Another best practice is to avoid future tense and keep everything in present tense to keep the reader "in the moment." "Below is an example output:" is less wordy than the original.

Reply:
Sure, but I think the problem here is that the user will not see the same output so there should be some kind of phrasing that will need to be future tense. I think there needs to be something to communicate that this output is similar to what the user will see from the above command. This is more meant to make sure the user is prepared for what to expect.

Reply:
I get what you mean, though I think "example output" is very clear that it's just an example and the user will obviously not see that exact output.

Reply:
I think we should assume nothing is obvious, especially if a user is just getting started. I think we at all times want to make sure the user has the ability to check herself or himself to make sure everything is going smoothly.